config/vendor-api-v1.yml

```diff
@@ -746,6 +747,12 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Qualification"
+        missing_gcses_explanation:
+          type: string
+          nullable: true
+          maxLength: 717 # 700 words: 3 x 200 (one per required GCSE) + 100 for our boilerplate
+          description: If the candidate lacks any required GCSEs, this field will contain their free-text explanation of why this is the case.
+          example: 'Maths GCSE or equivalent: I will take an exam with equivalencytesting.com on 18th August 2019'
```

Comment:
TIL of this site, looks like a real company. By using it as our example, could it be interpreted as advertising for them? Wondering if that's OK or if they're a household name like UCAS so it wouldn't be noticed by anyone.

This is a nit, and can be ignored.

Reply:
Good question. I got this from the designers. I'll replace it with some likely-sounding but fake domain name